packages/acceptance-tests/pkg-tests-specs/sources/commands/constraints.test.js

```diff
@@ -110,7 +111,7 @@ describe(`Commands`, () => {
                 ({code, stdout, stderr} = error);
               }
 
-              stdout = stdout.replace(/[^( ]+[\\/](yarn\.config)/g, `/path/to/$1`);
+              stdout = stdout.replace(new RegExp(npath.join(npath.fromPortablePath(path), `yarn.config.js`).replace(/\\/g, `\\\\`), `g`), `/path/to/yarn.config.js`);
```

Comment:
I'm a bit worried about turning a path into a regexp ... it'll also fail if there are `(){}[]^$+?*` in the path, for example

Reply:
Fixed